sys/dev/ath/if_athvar.h

```diff
@@ -607,6 +607,8 @@ struct ath_softc {
 	char			sc_tx_mtx_name[32];
 	struct mtx		sc_tx_ic_mtx;	/* TX queue mutex */
 	char			sc_tx_ic_mtx_name[32];
+	struct mtx		sc_usb_mtx;	/* USB access mutex */
```

Comment:
`ath_softc *sc = device_get_softc(self);` -> Can we modify the `ath_softc?` structure? I don't know this mechanism.